docs/devguide/flashfirmware.rst

```diff
@@ -1,58 +1,113 @@
 .. _flashfirmware:
 
-=================
-Flashing Firmware
-=================
+========================
+Build and flash firmware
+========================
 
-Building firmware
------------------
-Use yotta to build.
+micro:bit V1
+============
 
-Use target bbc-microbit-classic-gcc-nosd::
+This applies to MicroPython for the micro:bit V1, the source of which can be 
+found here: `bbcmicrobit/micropython <https://github.com/bbcmicrobit/micropython>`_.
 
-  yt target bbc-microbit-classic-gcc-nosd
+Dependencies
+------------
+- `cMake <https://cmake.org/>`_
+- `Arm gcc <https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads>`_
+- `git <https://git-scm.com/>`_
+- `ninja <https://ninja-build.org/>`_
+- `python <https://www.python.org/downloads/>`_
+- `srecord <http://srecord.sourceforge.net/>`_
+- `yotta <http://docs.yottabuild.org//>`_
 
-Run yotta update to fetch remote assets::
+The `yotta
+<http://docs.yottabuild.org//>`_ tool is used to build MicroPython, but before
+that takes place additional files have to be generated by the Makefile in
+preparation for the build, and additional data is added to the hex file after.
 
-  yt up
+Clone the repository and change directory to it::
+
+  $ git clone <https://github.com/bbcmicrobit/micropython
 
```

Comment:
```suggestion
  $ git clone https://github.com/bbcmicrobit/micropython
```

Reply:
done as part of https://github.com/bbcmicrobit/micropython/pull/711/commits/939efde1f837cd9aea60e972e3bb135b215baff6